index.js

```diff
@@ -34,9 +34,9 @@ exports.handler = (event, context, callback) => {
   }
   callback(null, response)
 }
-function findListenerSession(str) {
-  const param = (str || '').split('&').find(s => s.startsWith('ls='))
-  const id = (param || '').replace(/^ls=/, '')
+function findListenerEpisode(str) {
+  const param = (str || '').split('&').find(s => s.startsWith('le='))
```

Comment:
so this is parsing the query string? I see, it must not include the `?` so the split on `&` is in case there are multiple k=v pairs